backend/spec/features/admin/orders/adjustments_spec.rb

```diff
@@ -40,8 +41,11 @@
       end
     end
 
-    it "only shows eligible adjustments" do
-      expect(page).not_to have_content("ineligible")
```

Comment:
This page shows both eligible and non-eligible adjustments, and that's correct, because it highlights if they are not eligiblew by adding the class `adjustment-ineligible`. This test now checks that.